package.json

```diff
@@ -9,15 +9,17 @@
   },
   "keywords": [],
   "scripts": {
-    "build": "for PKG in packages/*; do echo Building $PKG...; tsc -P $PKG; done",
+    "build": "npm-run-all -c build:main build:packages",
+    "build:main": "tsc -p addon/ng2",
+    "build:packages": "for PKG in packages/*; do echo Building $PKG...; tsc -P $PKG; done",
     "test": "npm run test:packages && npm run test:cli",
     "mobile_test": "mocha tests/e2e/e2e_workflow.spec.js",
     "test:cli": "node tests/runner",
     "test:inspect": "node --inspect --debug-brk tests/runner",
     "test:packages": "node scripts/run-packages-spec.js",
     "build-config-interface": "dtsgen lib/config/schema.json --out lib/config/schema.d.ts",
     "eslint": "eslint .",
-    "tslint": "tslint \"**/*.ts\" -c tslint.json -e \"**/blueprints/*/files/**/*.ts\" -e \"node_modules/**\" -e \"tmp/**\"",
+    "tslint": "tslint '**/*.ts' -c tslint.json -e '**/blueprints/*/files/**/*.ts' -e 'node_modules/**' -e 'tmp/**' -e 'dist/**'",
```

Comment:
Revert the `\"` to `'` change please. It's a windows compatibility thing, it really needs to be `\"`.